tutorials/networking/high_level_multiplayer.rst

```diff
@@ -371,9 +372,151 @@ have loaded the game scene.
+        // It will be passed to every other peer.
+        // For example, the value of "name" can be set to something the player entered in a UI scene.
+        private readonly Dictionary<string, string> _playerInfo = new()
+        {
+            {"name", "Name"}
+        };
+
+        private int _playersLoaded = 0;
+
+        public override void _Ready()
+        {
+            Multiplayer.PeerConnected += _OnPlayerConnected;
+            Multiplayer.PeerDisconnected += _OnPlayerDisconnected;
+            Multiplayer.ConnectedToServer += _OnConnectedOk;
+            Multiplayer.ConnectionFailed += _OnConnectedFail;
+            Multiplayer.ServerDisconnected += _OnServerDisconnected;
+        }
+
+        public int JoinGame(string address = "")
+        {
+            if (string.IsNullOrEmpty(address))
+            {
+                address = _defaultServerIp;
+            }
+
+            ENetMultiplayerPeer peer = new();
+            Error error = peer.CreateClient(address, _port);
+            if (error != Error.Ok)
+            {
+                return (int)error;
+            }
+
+            Multiplayer.MultiplayerPeer = peer;
+            return (int)Error.Ok;
+        }
+
+        public int CreateGame()
+        {
+            ENetMultiplayerPeer peer = new();
+            Error error = peer.CreateServer(_port, _maxConnections);
+            if (error != Error.Ok)
+            {
+                return (int)error;
+            }
+
+            Multiplayer.MultiplayerPeer = peer;
+
+            _players[1] = _playerInfo;
+            EmitSignal(nameof(PlayerConnectedEventHandler), 1, _playerInfo);
+            return (int)Error.Ok;
+        }
+
+        public void RemoveMultiplayerPeer()
+        {
+            Multiplayer.MultiplayerPeer = null;
+        }
+
+        // When the server decides to start the game from a UI scene, do Lobby.load_game.rpc(filepath)
+        [Rpc(CallLocal = true, TransferMode = MultiplayerPeer.TransferModeEnum.Reliable)]
+        public void LoadGame(string gameScenePath)
+        {
+            GetTree().ChangeSceneToFile(gameScenePath);
+        }
+
+        // Every peer will call this when they have loaded the game scene.
+        [Rpc(mode: MultiplayerApi.RpcMode.AnyPeer, CallLocal = true, TransferMode = MultiplayerPeer.TransferModeEnum.Reliable)]
+        public void PlayerLoaded()
+        {
+            if (!Multiplayer.IsServer()) return;
+            _playersLoaded++;
+            if (_playersLoaded == _players.Count)
+            {
+                // TODO Replace "/root/Game.start_game()" with the equivalent C# method to start the game.
+                _playersLoaded = 0;
+            }
+        }
+
+        // When a peer connects, send them my player info.
+        // This allows transfer of all desired data for each player, not only the unique ID.
+        private void _OnPlayerConnected(long id)
```

Comment:
We don't usually prefix private method names with an underscore, it's against .NET naming guidelines. Same for the other methods.

```suggestion
        private void OnPlayerConnected(long id)
```